docs/introduction/why.rst

```diff
@@ -0,0 +1,21 @@
+.. _why_renku:
+
+Why Renku
+---------
+
+If you carry out a data science project, chances are you will be faced with
```

Comment:
```
data-science project
```

Reply:
i didn't change any `data-science` to `data science` because the dash looks weird to me but i suppose i could be convinced otherwise